src/lib/other/registerqappassociation.cpp

```diff
@@ -261,6 +267,65 @@ void RegisterQAppAssociation::registerAllAssociation()
     }
 }
 
+// The code of the following method was taken
+// from https://github.com/mozilla/gecko-dev/blob/master/browser/components/shell/nsWindowsShellService.cpp#L364
+// that is licensed under MPL-2.0.
+bool RegisterQAppAssociation::showNativeDefaultAppSettingsUi()
+{
+    if (!isVistaOrNewer()) {
+        return false;
+    }
+
+    if (isWin10OrNewer()) {
+        IApplicationActivationManager* pActivator;
+        HRESULT hr = CoCreateInstance(CLSID_ApplicationActivationManager,
+                                      nullptr,
+                                      CLSCTX_INPROC,
+                                      IID_IApplicationActivationManager,
+                                      (void**)&pActivator);
+
+        if (!SUCCEEDED(hr)) {
+            return false;
+        }
+
+        DWORD pid;
+        hr = pActivator->ActivateApplication(
+            L"windows.immersivecontrolpanel_cw5n1h2txyewy"
```

Comment:
The method [IApplicationActivationManager::ActivateApplication()](https://msdn.microsoft.com/en-us/library/windows/desktop/hh706903(v=vs.85).aspx) activates a specified Windows Store app  by its appUserModelId.
The appUserModelId of `Settings` is:
`windows.immersivecontrolpanel_cw5n1h2txyewy!microsoft.windows.immersivecontrolpanel`

Some other appUserModelIds are as follows:
```cpp
windows.immersivecontrolpanel_cw5n1h2txyewy!microsoft.windows.immersivecontrolpanel
Microsoft.MicrosoftEdge_8wekyb3d8bbwe!MicrosoftEdge
Microsoft.MicrosoftEdge_8wekyb3d8bbwe!ContentProcess
Microsoft.MicrosoftEdge_8wekyb3d8bbwe!BCHost
Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy!App
Microsoft.Windows.Photos_8wekyb3d8bbwe!App
```
As you see `cw5n1h2txyewy` and `8wekyb3d8bbwe` have been repeated. So I think they are somethings like a product-category id for a company registered in Windows store.